modules/howtos/examples/kv_operations.rb

```diff
@@ -92,6 +92,13 @@
   puts "Example requires a multi-node environment"
 end
 
+begin
+  # tag::durability-observed[]
+  collection.upsert("my-document", {"doc" => true}, 
+                  Options::Upsert(persist_to: :none, replicate_to: :two))
+  # end::durability-observed[]
+rescue Error::DurabilityImpossible
```

Comment:
Seems we will have Legacy Durability in Ruby SDK 3.4.

@avsej I'm just prepping this example before the release, let me know if anything looks incorrect. I simply tried to match the example we have in [Java](https://github.com/couchbase/docs-sdk-java/blob/d3c6d9cd0b3b9e8aa5d4a785da7f2cd73c2fc5c2/modules/howtos/examples/KvOperations.java#L181-L182).